prettier.config.js

```diff
@@ -1,8 +1,11 @@
-const config = {
+import { loadPrettierPlugins } from './src/utils/loadPrettierPlugins.js'
+
+const getConfig = async () => ({
   printWidth: 100,
   singleQuote: true,
   arrowParens: 'avoid',
   semi: false,
+  plugins: await loadPrettierPlugins(),
   overrides: [
```

Comment:
```suggestion
export default {
  printWidth: 100,
  singleQuote: true,
  arrowParens: 'avoid',
  semi: false,
  plugins: loadPrettierPlugins(),
  overrides: [
```

Defining a function to be immediately called is not required